packages/nextcloud/lib/src/version_supported.dart

```diff
@@ -1,33 +1,57 @@
 part of '../nextcloud.dart';
 
+/// Version of core/Server supported
+const coreSupportedVersion = 26;
+
+/// API version of the news app supported
+const newsSupportedVersion = 'v1-3';
+
+/// API version of the notes app supported
+const notesSupportedVersion = 1;
+
+/// All supported versions
+const supportedVersions = [
```

Comment:
Where do we need this?

I don't think a list is that helpful as here the app name is relevant. `Maybe a Map<appName, versionSupported>` ?

Reply:
I noticed it too, removed it.